ext/lib/crypto/mbedtls/configs/config-tls-benchmark.h

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Configuration for TLS Benchmark
+ */
+
```

Comment:
I looked into it.  And though of trying to enable TLS_USER_CONFIG_ENABLE and defines extra configs in TLS_USER_CONFIG_FILE. But I have to enable many like this in prj.conf file
TLS_ECP_DP_SECP192R1_ENABLED
TLS_ECP_DP_SECP224R1_ENABLED
TLS_ECP_DP_SECP256R1_ENABLED
TLS_ECP_DP_SECP384R1_ENABLED
TLS_ECP_DP_SECP521R1_ENABLED
TLS_ECP_DP_SECP192K1_ENABLED
TLS_ECP_DP_SECP224K1_ENABLED
TLS_ECP_DP_SECP256K1_ENABLED
TLS_ECP_DP_BP256R1_ENABLED
TLS_ECP_DP_BP384R1_ENABLED
TLS_ECP_DP_BP512R1_ENABLED
TLS_ECP_DP_CURVE25519_ENABLED
TLS_ECP_DP_CURVE448_ENABLED
and etc....

Which I want to avoid. Just plain simple minimal configuration of MBEDTLS in prj.conf which I submitted now.

Reply:
I also agree that "multiplying these config files on a per sample basis is not a good approach". :)

Reply:
Yes, that's true that in order to enable everything (which makes sense for benchmark app) quite a lot of configs need to set. But the fact is, that we do exactly the same in custom mbedtls configuration, just in different form. And with this approach we lose flexibility of enabling/disabling specific algorithms to benchmark through Kconfig.

But I get your point, that enabling "everything" requires a lot of configs to remember and set. So perhaps, as an improvement we could introduce a master switch, for instance per alogrithm family, to enable all algorithms in specific group? Currently, there are 4 groups distinguished in Kconfig (Key Exchange, Curves, Cipher and MAC). This would significantly reduce number of configs to set in the `prj.conf` file.